### DISCUSSION TITLE
[FEATURE REQUEST] Make a Liberapay or another recurrent donations platform profile for the most active contributors in EddieHub

### DISCUSSION THREAD

**Discussion by Panquesito7** [EddieHubCommunity/support#2278] [Ideas]
### Description of feature request, idea

Make a Liberapay or another recurrent donations platform profile for the most active contributors in EddieHub,

### Add any other context about the feature request here

In Liberapay, we can create a team for EddieHub. For example: if someone donates to us $10 per week, and there are 10 members, each member will receive a dollar. A lot of people are working very hard in EddieHub, and they're not receiving any money for it. Just a thought.

[a comment by an automated account is not shown]

**Comment by eddiejaoude** [comment]
Thank you for the suggestion. This is a very difficult subject to solve, I love the idea of the active community members getting more benefits for their work they do in the community. But I think at the moment it is not the right time because there are no donations, for example I do not get that many on GitHub Sponsors, plus I think I would prefer the donations to come from companies rather than the community members themselves.

We have a long list of things we need to do for the EddieHub community first, but we can leave this issue open, to continue to bounce ideas on the subject.

**Comment by Vyvy-vi** [comment]
Maybe we can turn this into a discussion so that it doesn't get lost?
